src/components/DragBox/DragBox.js

```diff
@@ -311,10 +323,14 @@ class DragBox extends React.Component {
   render() {
     const { refDragElement, onMouseDown, renderRect, renderContents } = this;
     return (
-      <div ref={refDragElement} onMouseDown={onMouseDown}>
+      <DragBoxDiv
+        ref={refDragElement}
+        onMouseDown={onMouseDown}
+        disablePointerEvents={this.shouldDrawRect(this.calcRect())}
```

Comment:
This seems a little wasteful since the same functions are being called within `renderRect`. Could `renderRect` be updated to accept a rectangle calculated in the scope of this function?